<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CargaWindow</class>
 <widget class="QMainWindow" name="CargaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget {
border-image:url(:/imagen/fondoMalv.jpg);
background-repeat : no-repeat;
border:5px solid black;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="setCantBarc">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="setTamMatr">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#centralwidget {
border-image: url(:/imagen/fondoMalv.jpg);
background-repeat : no-repeat;
border:5px solid black;
}</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxAleatorios">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0.3);</string>
    </property>
    <property name="text">
     <string>Aleatorios</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCantBarcos">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0.3);
</string>
    </property>
    <property name="text">
     <string>Elija cantidad de barcos</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTamMatr">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0.3);</string>
    </property>
    <property name="text">
     <string>Elija tamanio Matriz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonJugar">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>120</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <italic>true</italic>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #000080;
color: white;</string>
    </property>
    <property name="text">
     <string>JOGA</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCargarPartida">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>CARGAR PARTIDA</string>
    </property>
   </widget>
   <widget class="QLabel" name="Titulo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>701</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>38</pointsize>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-shadow: 0 0 3px #ff0000;</string>
    </property>
    <property name="text">
     <string>BATALLA NAVAL MALVINAS</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
